server/data/data-library/src/main/java/org/apache/james/mailrepository/lib/AbstractMailRepository.java

```diff
@@ -141,6 +141,7 @@ public void remove(String key) throws MessagingException {
         if (lock(key)) {
             try {
                 internalRemove(key);
+            } catch (MessagingException e) {
```

Comment:
It was to fix the test in error (AbstractMailRepository.testEmptyRepository): removing an unexisting message should not throw.